README.md

```diff
@@ -1,30 +1,22 @@
-<!--===- README.md
+# Flang
 
-   Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
-   See https://llvm.org/LICENSE.txt for license information.
-   SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-
--->
-
-# F18
-
-F18 is a ground-up implementation of a Fortran front end written in modern C++.
-F18, when combined with LLVM, is intended to replace the Flang compiler.
-
-Flang is a Fortran compiler targeting LLVM.
-Visit the [Flang wiki](https://github.com/flang-compiler/flang/wiki)
-for more information about Flang.
+Flang is a ground-up implementation of a Fortran front end written in modern
+C++. It started off as the f18 project (https://github.com/flang-compiler/f18)
+at Nvidia with an aim to replace the old flang project
```

Comment:
I would say "previous flang project" :-)

Reply:
No need to call out NVIDIA, but if you do, please also include the NNSA.

Reply:
I've changed it to not call out anyone in particular, let me know if the new wording is ok!